specification/metrics/listcost.md

```diff
@@ -0,0 +1,34 @@
+# List Cost
+
+The List Cost represents a charge inclusive of the impacts of volume/tier-based rates while excluding any negotiated or commitment-based discounts. This cost is denominated in the [Billing Currency](#billingcurrency). The List Cost equals the [BilledCost](#billedcost) if there are no negotiated or commitment-based discounts against the resource and/or service. By comparing it with [EffectiveCost](#effectivecost) you can calculate savings based on rate optimization activities.
```

Comment:
I believe that List Cost should also exclude Spot instances as well, so practitioners could use List price to calculate savings from leveraging spot. 
Suggested change of definition: while excluding negotiated, commitment-based or spare capacity (spot) discounts

Reply:
This seems like a good suggestion @ijurica  happy for me to change it per Yuriys suggestion?

Reply:
suggestion: 
...while excluding any negotiated, commitment-based discounts or spare capacity (spot) discounts...

Reply:
To include the capability to calculate savings from utilizing spare capacity (spot) discounts, we are still exploring a couple of alternative List Cost definitions.

- OPTION A (current version): _The List Cost represents a charge inclusive of the impacts of volume/tier-based rates while excluding any negotiated discounts, commitment-based discounts or the amortization of upfront charges (one-time or recurring)._

- OPTION B : _The List Cost represents a charge inclusive of the impacts of volume/tier-based rates while excluding any negotiated discounts, commitment-based discounts, **spare capacity (spot) discounts** or the amortization of upfront charges (one-time or recurring)._

- OPTION C (aligned with pricing model column terminology): _The List Cost represents a charge inclusive of the impacts of volume/tier-based rates while excluding any negotiated discounts, commitment-based discounts, **the impacts of dynamic pricing model** or the amortization of upfront charges (one-time or recurring)._

Note: **Alignment of all unit price and cost metric definitions is implied.**

Reply:
I think Option B looks the most correct. Its very clear and it mentions the exclusion of the amortization of upfront charges which I think is key to exclude for the List Cost.

Reply:
Option B is crystal clear

Reply:
I like Option C over Option B because it is more vendor-neutral.  Removing `(spot)` from Option B would make me more in favor because GCP's name for `spot` is `preemptible`.

Reply:
Discussed at the meeting and decided to simplify the definition (similar to ListUnitPrice).
Notes regarding included/excluded impacts are provided in the supporting content